packages/primitives/src/Svg/Svg.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from '@emotion/styled'
+import { Theme, css } from 'theme-ui'
+import { SerializedStyles } from '@emotion/serialize'
+import {
+  StyledSystemProps,
+  variant,
+  sx,
+  shouldForwardProp,
+  styledSystemProps,
+} from '../Box/Box'
+
+/**
+ * base
+ *
+ * Generate the base CSS filled by the currentColor by default
+ *
+ * @param props
+ */
+export const baseStyles = ({ theme }: { theme: Theme }): SerializedStyles =>
+  css({
+    fill: 'currentColor',
+    color: 'black',
```

Comment:
@alexanderchan go ahead and add `display: block` to this because of SVG's weird positioning.